advanced/button-interrupt/exercise/src/main.rs

```diff
@@ -0,0 +1,70 @@
+// reference:
+// https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/freertos.html
+
+use std::sync::mpsc;
```

Comment:
Once clippy is happy with the imports in the solution, I would make this the same. Or at least remove `std::sync::mpsc`, which is redundant.